Makefile

```diff
@@ -6,7 +6,7 @@ MDOX_VALIDATE_CONFIG ?= .mdox.validate.yaml
 # if macos, use gsed
 SED ?= $(shell which gsed 2>/dev/null || which sed)
 
-LINT_DIRS = $(shell go list ./... | grep -v "internal/prometheus")
+LINT_DIRS = $(shell go list ./... | grep -v "parser")
```

Comment:
Maybe we want to enable regexps and match for `^parser` to make sure we don't accidentally exclude some nested packages.